examples/aggregation/aggregation.md

```diff
@@ -0,0 +1,21 @@
+# Aggregation
```

Comment:
I think it is better to give a small description on what is meant by grouping/non grouping keys and sequence variables